### PR TITLE
COM-19893: Implement the Download Section

### PR DESCRIPTION
**Pull request by SylvainGuittard**
jira issue: https://jira.ez.no/browse/COM-19893

Improvements for mobile devices needed!

# Desktop - Composer part
ezplatform-com-after-composer

# Desktop - Download part
ezplatform-com-after-download

### CODE REVIEW

**Comment by SylvainGuittard** [comment]
ping @clash82 @damianz5

**Comment by damianz5** [comment]
+1

**Comment by SylvainGuittard** [comment]
Rebased! Ready for your review @clash82 ;)

**Comment by clash82** [comment]
Rebased? I still see two commits here :)

**Comment by SylvainGuittard** [comment]
Rebased and only one commit!

File: app/Resources/views/row/download_section.html.twig [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                                {% endif %}
+
+                                {% for release in releases %}
+                                    <tr class="releases">
+                                        <th scope="row">
+                                            <a href="{{ ez_field_value(release, 'file') }}">{{ ez_render_field(release, 'release_version') }}</a>
+                                        </th>
+                                        <td class="file">
+                                            <a href="{{ ez_field_value(release, 'file') }}">
+                                                {{ release.fields.file["eng-GB"].fileName }}
```
Comment (clash82) [inline app/Resources/views/row/download_section.html.twig]: a bit shorter version: `release.fields.file.eng-GB.fileName`

File: app/config/ezplatform.yml [hunk 2]
```diff
@@ -45,3 +45,8 @@ ezpublish:
                         template: row\banner_slider.html.twig
                         match:
                             Identifier\ContentType: banner_slider
+                    download_section:
+                        template: row\download_section.html.twig
+                        controller: AppBundle:Download:getFiles
```
Comment (clash82) [inline app/config/ezplatform.yml]: for view controllers you should use `show` prefix, `get` is more related to objects

File: src/AppBundle/Controller/DownloadController.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+class DownloadController extends Controller
+{
+    /**
+     * This method will return all the releases, sub-items of Releases and Betas content objects
+     *
+     * @param \eZ\Publish\Core\MVC\Symfony\View\ContentView $view
+     *
+     * @return \eZ\Publish\Core\MVC\Symfony\View\ContentView
+     *
```
Comment (clash82) [inline src/AppBundle/Controller/DownloadController.php]: empty line

File: src/AppBundle/Controller/DownloadController.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+use eZ\Publish\API\Repository\Values\Content\LocationQuery;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class DownloadController extends Controller
+{
+    /**
+     * This method will return all the releases, sub-items of Releases and Betas content objects
```
Comment (clash82) [inline src/AppBundle/Controller/DownloadController.php]: Missing dot at the end of the string ;)

File: app/Resources/views/row/download_section.html.twig [hunk 5]
```diff
@@ -0,0 +1,150 @@
+<section id="Download">
```
Comment (clash82) [inline app/Resources/views/row/download_section.html.twig]: cs: always use lowercase for ids and classes

File: web/assets/css/download.css [hunk 6]
```diff
@@ -0,0 +1,346 @@
...
+
+section#Download div.details p {
+    font-size: 15px;
+}
+
+section#Download .nav-pills>li.active>a,
+section#Download .nav-pills>li.active>a:focus,
+section#Download .nav-pills>li.active>a:hover {
+    color: #fff;
+    background-color: #f7f5f2;
```
Comment (clash82) [inline web/assets/css/download.css]: `background`

File: web/assets/css/download.css [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+}
+
+section#Download .window .help {
+    color: #fff;
+    margin-top: 10px;
+    margin-bottom: 30px;
+}
+
+section#Download .titlebar {
+    background-color: #e3e5e5;
```
Comment (clash82) [inline web/assets/css/download.css]: `background`

File: web/assets/css/download.css [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+    width: 100%;
+    height: 35px;
+    border-top: none;
+    border-bottom: none;
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+    user-select: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
```
Comment (clash82) [inline web/assets/css/download.css]: skip

File: web/assets/css/download.css [hunk 9]
```diff
@@ -0,0 +1,346 @@
...
+    text-align: center;
+    width: 100%;
+    height: 35px;
+    border-top: none;
+    border-bottom: none;
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+    user-select: none;
+    -webkit-user-select: none;
+    -moz-user-select: none;
```
Comment (clash82) [inline web/assets/css/download.css]: skip

File: web/assets/css/download.css [hunk 10]
```diff
@@ -0,0 +1,346 @@
...
+section#Download .window h3 {
+    margin-top: 0;
+}
+
+section#Download .download-table {
+    margin-top: 40px;
+}
+
+section#Download .download-table thead th {
+    background-color: #f15a22;
```
Comment (clash82) [inline web/assets/css/download.css]: `background`

File: web/assets/css/download.css [hunk 11]
```diff
@@ -0,0 +1,346 @@
...
+    display: block;
+    line-height: 80px;
+    color: #fff;
+    text-decoration: none;
+}
+
+section#Download .download-table tr.beta td,
+section#Download .download-table tr.beta th {
+    height: 80px;
+    background-color: #f7941d;
```
Comment (clash82) [inline web/assets/css/download.css]: `background`

File: web/assets/css/download.css [hunk 12]
```diff
@@ -0,0 +1,346 @@
...
+    width: 35px;
+}
+
+section#Download .download-table tr.releases {
+    border-left: 1px solid #ccc;
+    border-right: 1px solid #ccc;
+}
+
+section#Download .download-table.table-striped>tbody>tr:nth-of-type(odd) {
+    background-color: #f7f5f2;
```
Comment (clash82) [inline web/assets/css/download.css]: `background`

File: web/assets/images/download.svg [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+			<path class="st1" d="M53.7,32.3c-0.1,0.1-0.3,0.4-0.4,0.9l-1,4c-0.5,1.9-2.3,3.5-4.4,3.5H36.1c-2,0-3.9-1.5-4.4-3.5l-1-4
+				c-0.1-0.4-0.3-0.7-0.4-0.9H16.1l0,0v23.1c0,1.1,0.8,1.8,1.8,1.8h48.2c1,0,1.8-0.9,1.8-1.8V32.3H53.7z"/>
+		</g>
+		<path class="st2" d="M53.6,30.3h14.3l-8.1-13.8c-0.2-0.4-0.7-0.7-1.2-0.7h-9l-6,8l-4.9-0.9L34,15.7h-8.8c-0.5,0-0.9,0.3-1.2,0.7
+			l-7.9,13.9h14.3c1.1,0,1.8,1.2,2.1,2.2l1,4c0.3,1.2,1.4,2,2.5,2h11.8c1.1,0,2.2-0.9,2.5-2l1-4C51.7,31.5,52.4,30.3,53.6,30.3z"/>
+	</g>
+	<path class="st0" d="M52.2,15.9c-1-1-3.5-0.8-4.5,0.2l-4.2,4.2V4c0-1.4-0.7-2-2-2c-1.4,0-2,0.7-2,2v16.2l-4-4.4c-1-1-2.6-1-3.6,0
+		s-1,2.6,0,3.6l8.1,8c0.5,0.7,1.3,1.2,2.1,1.2s1.7-0.5,2.1-1.2l8.1-8C53.2,18.5,53.2,16.9,52.2,15.9z"/>
+</g>
+</svg>
```
Comment (clash82) [inline web/assets/images/download.svg]: same image minified (using SVG minifier):

```xml
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 84 65"><style type="text/css">
	.st0{fill:#514744;}
	.st1{fill:#FCB812;}
	.st2{fill:#F15A22;}
</style><path class="st0" d="M70.3 63H13.7c-1.8 0-3.3-1.6-3.3-3.4V30.3c0-0.1 0-0.1 0-0.2 0-0.3 0.2-1.1 0.5-1.6l9.8-15.9c0.6-1 1.7-1.7 2.8-1.7H35c0.7 0 1.2 0.5 1.2 1.2s-0.6 1.3-1.3 1.3H23.4c-0.3 0-0.6 0.2-0.8 0.5l-9.7 15.9c0 0.1-0.1 0.4-0.1 0.6v29.2c0 0.6 0.4 1.1 0.9 1.1h56.6c0.5 0 0.9-0.5 0.9-1.1V30.4c0-0.3-0.1-0.6-0.2-0.7L61.4 14c-0.1-0.3-0.4-0.5-0.8-0.5H49c-0.7 0-1.2-0.5-1.2-1.2s0.5-1.2 1.2-1.2h11.6c1.2 0 2.2 0.6 2.8 1.7l9.7 15.8c0.3 0.5 0.4 1.3 0.5 1.6 0 0.1 0 0.1 0 0.2v29.3C73.6 61.4 72.2 63 70.3 63z"/><path class="st1" d="M53.7 32.3c-0.1 0.1-0.3 0.4-0.4 0.9l-1 4c-0.5 1.9-2.3 3.5-4.4 3.5H36.1c-2 0-3.9-1.5-4.4-3.5l-1-4c-0.1-0.4-0.3-0.7-0.4-0.9H16.1l0 0v23.1c0 1.1 0.8 1.8 1.8 1.8h48.2c1 0 1.8-0.9 1.8-1.8V32.3H53.7z"/><path class="st2" d="M53.6 30.3h14.3l-8.1-13.8c-0.2-0.4-0.7-0.7-1.2-0.7h-9l-6 8 -4.9-0.9L34 15.7h-8.8c-0.5 0-0.9 0.3-1.2 0.7l-7.9 13.9h14.3c1.1 0 1.8 1.2 2.1 2.2l1 4c0.3 1.2 1.4 2 2.5 2h11.8c1.1 0 2.2-0.9 2.5-2l1-4C51.7 31.5 52.4 30.3 53.6 30.3z"/><path class="st0" d="M52.2 15.9c-1-1-3.5-0.8-4.5 0.2l-4.2 4.2V4c0-1.4-0.7-2-2-2 -1.4 0-2 0.7-2 2v16.2l-4-4.4c-1-1-2.6-1-3.6 0s-1 2.6 0 3.6l8.1 8c0.5 0.7 1.3 1.2 2.1 1.2s1.7-0.5 2.1-1.2l8.1-8C53.2 18.5 53.2 16.9 52.2 15.9z"/></svg>
```

File: web/assets/images/download-arrow.svg [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+		<path class="st0" d="M18.5,3C27,3,34,10,34,18.5S27,34,18.5,34S3,27,3,18.5S10,3,18.5,3 M18.5,1C8.8,1,1,8.8,1,18.5
+			S8.8,36,18.5,36S36,28.2,36,18.5S28.2,1,18.5,1L18.5,1z"/>
+	</g>
+	<g id="Shape_39729_copy_2">
+		<path class="st1" d="M24.2,18.7c-0.6-0.5-1.9-0.4-2.5,0.1L19.4,21v-7.4c0-0.7-0.4-1.1-1.2-1.1c-0.8,0-1.2,0.4-1.2,1.1V21l-2.2-2.3
+			c-0.6-0.5-1.5-0.5-2,0c-0.6,0.5-0.6,1.4,0,1.9l4.5,4.3c0.3,0.4,0.7,0.6,1.2,0.6c0.5,0,0.9-0.2,1.2-0.6l4.5-4.3
+			C24.7,20.1,24.8,19.2,24.2,18.7z"/>
+	</g>
+</g>
+</svg>
```
Comment (clash82) [inline web/assets/images/download-arrow.svg]: minified version:

```xml
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 37 37"><style type="text/css">
	.st0{fill:#fff;}
	.st1{fill-rule:evenodd; clip-rule:evenodd; fill:#fff;}
</style><g><path class="st0" d="M18.5 3C27 3 34 10 34 18.5S27 34 18.5 34 3 27 3 18.5 10 3 18.5 3M18.5 1C8.8 1 1 8.8 1 18.5S8.8 36 18.5 36 36 28.2 36 18.5 28.2 1 18.5 1L18.5 1z"/></g><g id="Shape_39729_copy_2"><path class="st1" d="M24.2 18.7c-0.6-0.5-1.9-0.4-2.5 0.1L19.4 21v-7.4c0-0.7-0.4-1.1-1.2-1.1 -0.8 0-1.2 0.4-1.2 1.1V21l-2.2-2.3c-0.6-0.5-1.5-0.5-2 0 -0.6 0.5-0.6 1.4 0 1.9l4.5 4.3c0.3 0.4 0.7 0.6 1.2 0.6 0.5 0 0.9-0.2 1.2-0.6l4.5-4.3C24.7 20.1 24.8 19.2 24.2 18.7z"/></g></svg>
```

File: web/assets/images/composer.svg [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+	<g>
+		<path class="st4" d="M73,40h-5c-0.8,0-1.4,0.6-1.4,1.4c0,0.8,0.6,1.4,1.4,1.4h5c0.8,0,1.4-0.6,1.4-1.4S73.8,40,73,40z M63.2,40H61
+			c-0.8,0-1.4,0.6-1.4,1.4c0,0.8,0.6,1.4,1.4,1.4h2.1c0.8,0,1.4-0.6,1.4-1.4C64.6,40.6,63.9,40,63.2,40z M73,35.8h-2.9
+			c-0.8,0-1.4,0.6-1.4,1.4c0,0.8,0.6,1.4,1.4,1.4H73c0.8,0,1.4-0.6,1.4-1.4C74.4,36.4,73.8,35.8,73,35.8z M67.4,37.2
+			c0-0.8-0.6-1.4-1.4-1.4h-0.7c-0.8,0-1.4,0.6-1.4,1.4c0,0.8,0.6,1.4,1.4,1.4H66C66.7,38.6,67.4,37.9,67.4,37.2z M73,31.5h-7.8
+			c-0.8,0-1.4,0.6-1.4,1.4l0,0c0,0.8,0.6,1.4,1.4,1.4H73c0.8,0,1.4-0.6,1.4-1.4l0,0C74.4,32.1,73.8,31.5,73,31.5z M61,34.3
+			c0.8,0,1.4-0.6,1.4-1.4l0,0c0-0.8-0.6-1.4-1.4-1.4c-0.8,0-1.4,0.6-1.4,1.4l0,0C59.6,33.7,60.3,34.3,61,34.3z"/>
+	</g>
+</g>
+</svg>
```
Comment (clash82) [inline web/assets/images/composer.svg]: minified version:

```xml
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 84 65"><style type="text/css">
	.st0{fill:#FFFFFF;}
	.st1{fill:#514744;}
	.st2{fill:#2B85B0;}
	.st3{fill:#FCB812;}
	.st4{fill:#F15A22;}
	.st5{fill:#D13100;}
	.st6{fill:#FEBB31;}
	.st7{fill:#71D109;}
</style><path class="st0" d="M9.2 61.8c-3.3 0-5.9-2.6-5.9-5.9V13.7c0-0.7 0.6-1.2 1.2-1.2h63.2c0.7 0 1.2 0.6 1.2 1.2v42.2c0 3.2-2.7 5.9-5.9 5.9C63 61.8 9.2 61.8 9.2 61.8z"/><path class="st1" d="M67.6 13.7v42.2c0 2.6-2.1 4.6-4.7 4.6H9.2c-2.6 0-4.7-2.1-4.7-4.6V13.7H67.6M67.6 11.3H4.5c-1.4 0-2.5 1.1-2.5 2.5V56c0 3.8 3.2 7 7.2 7H63c3.9 0 7.2-3.2 7.2-7.1V13.7C70.1 12.4 69 11.3 67.6 11.3L67.6 11.3z"/><path class="st2" d="M12.7 27.1h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H12.7c-1 0-1.9 0.8-1.9 1.9l0 0C10.8 26.3 11.7 27.1 12.7 27.1z"/><path class="st3" d="M31.2 27.1h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H31.2c-1 0-1.9 0.8-1.9 1.9l0 0C29.4 26.3 30.2 27.1 31.2 27.1z"/><path class="st2" d="M20.6 44.3h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H20.6c-1 0-1.9 0.8-1.9 1.9l0 0C18.8 43.5 19.6 44.3 20.6 44.3z"/><path class="st3" d="M39.2 44.3h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H39.2c-1 0-1.9 0.8-1.9 1.9l0 0C37.3 43.5 38.2 44.3 39.2 44.3z"/><path class="st4" d="M32.2 36.2h16.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H32.2c-1 0-1.9 0.8-1.9 1.9l0 0C30.3 35.4 31.2 36.2 32.2 36.2z"/><path class="st3" d="M20.6 36.2h4.9c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9h-4.9c-1 0-1.9 0.8-1.9 1.9l0 0C18.8 35.4 19.6 36.2 20.6 36.2z"/><path class="st2" d="M56.1 36.2h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H56.1c-1 0-1.9 0.8-1.9 1.9l0 0C54.3 35.4 55.1 36.2 56.1 36.2z"/><path class="st4" d="M12.4 53.3h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H12.4c-1 0-1.9 0.8-1.9 1.9l0 0C10.6 52.5 11.4 53.3 12.4 53.3z"/><path class="st3" d="M32.4 53.3h12.1c1 0 1.9-0.8 1.9-1.9l0 0c0-1-0.8-1.9-1.9-1.9H32.4c-1 0-1.9 0.8-1.9 1.9l0 0C30.5 52.5 31.3 53.3 32.4 53.3z"/><path class="st4" d="M4.5 14.9c-0.7 0-1.2-0.6-1.2-1.2V6.9c0-2 1.7-3.7 3.7-3.7h58.2c2 0 3.7 1.6 3.7 3.7v6.8c0 0.7-0.6 1.2-1.2 1.2C67.7 14.9 4.5 14.9 4.5 14.9z"/><path class="st1" d="M65.2 4.4c1.4 0 2.5 1.1 2.5 2.4v6.8H4.5V6.9c0-1.3 1.1-2.4 2.5-2.4L65.2 4.4M65.2 2H6.9C4.2 2 2 4.2 2 6.9v6.8c0 1.4 1.1 2.5 2.5 2.5h63.2c1.4 0 2.5-1.1 2.5-2.5V6.9C70.1 4.2 67.9 2 65.2 2L65.2 2z"/><path class="st5" d="M8.7 7.6L8.7 7.6c-0.8 0-1.4 0.6-1.4 1.3V9c0 0.7 0.6 1.3 1.3 1.3h0.1C9.4 10.4 10 9.8 10 9V8.9C10 8.2 9.4 7.6 8.7 7.6"/><path class="st6" d="M12.7 7.6h-0.1c-0.7 0-1.3 0.6-1.3 1.3V9c0 0.7 0.6 1.3 1.3 1.3h0.1c0.7 0 1.3-0.6 1.3-1.3V8.9C14.1 8.2 13.5 7.6 12.7 7.6"/><path class="st7" d="M16.8 7.6h-0.1c-0.7 0-1.3 0.6-1.3 1.3V9c0 0.7 0.6 1.3 1.3 1.3h0.1c0.7 0 1.3-0.6 1.3-1.3V8.9C18.1 8.2 17.5 7.6 16.8 7.6"/><path class="st0" d="M62.5 7.5H21.3c-0.3 0-0.6 0.3-0.6 0.6V10c0 0.3 0.3 0.6 0.6 0.6h41.2c0.3 0 0.6-0.3 0.6-0.6V8.1C63 7.8 62.8 7.5 62.5 7.5"/><path class="st0" d="M58.7 25.2h18.8c1.8 0 3.3 1.5 3.3 3.3v17.4c0 1.8-1.5 3.3-3.3 3.3H58.7c-1.8 0-3.3-1.5-3.3-3.3V28.5C55.4 26.6 56.9 25.2 58.7 25.2z"/><path class="st1" d="M77.5 50.4H58.7c-2.5 0-4.5-2-4.5-4.5V28.5c0-2.5 2-4.5 4.5-4.5h18.8c2.5 0 4.5 2 4.5 4.5v17.4C82 48.3 80 50.4 77.5 50.4zM58.7 26.4c-1.1 0-2.1 0.9-2.1 2.1v17.4c0 1.1 0.9 2.1 2.1 2.1h18.8c1.1 0 2.1-0.9 2.1-2.1V28.5c0-1.1-0.9-2.1-2.1-2.1H58.7z"/><path class="st4" d="M73 40h-5c-0.8 0-1.4 0.6-1.4 1.4 0 0.8 0.6 1.4 1.4 1.4h5c0.8 0 1.4-0.6 1.4-1.4S73.8 40 73 40zM63.2 40H61c-0.8 0-1.4 0.6-1.4 1.4 0 0.8 0.6 1.4 1.4 1.4h2.1c0.8 0 1.4-0.6 1.4-1.4C64.6 40.6 63.9 40 63.2 40zM73 35.8h-2.9c-0.8 0-1.4 0.6-1.4 1.4 0 0.8 0.6 1.4 1.4 1.4H73c0.8 0 1.4-0.6 1.4-1.4C74.4 36.4 73.8 35.8 73 35.8zM67.4 37.2c0-0.8-0.6-1.4-1.4-1.4h-0.7c-0.8 0-1.4 0.6-1.4 1.4 0 0.8 0.6 1.4 1.4 1.4H66C66.7 38.6 67.4 37.9 67.4 37.2zM73 31.5h-7.8c-0.8 0-1.4 0.6-1.4 1.4l0 0c0 0.8 0.6 1.4 1.4 1.4H73c0.8 0 1.4-0.6 1.4-1.4l0 0C74.4 32.1 73.8 31.5 73 31.5zM61 34.3c0.8 0 1.4-0.6 1.4-1.4l0 0c0-0.8-0.6-1.4-1.4-1.4 -0.8 0-1.4 0.6-1.4 1.4l0 0C59.6 33.7 60.3 34.3 61 34.3z"/></svg>
```

File: app/config/config_ezplatform_com.yml [hunk 16]
```diff
@@ -1,2 +1,4 @@
 parameters:
     layout_id: 54
+    release_container_location_id: 57
```
Comment (clash82) [inline app/config/config_ezplatform_com.yml]: you could add some comments here to describe each parameter

File: app/Resources/views/row/download_section.html.twig (hunk 1 above)
Comment (SylvainGuittard) [inline app/Resources/views/row/download_section.html.twig]: No, you need it.
```Key "eng" for array with keys "eng-GB" does not exist```

File: app/config/ezplatform.yml (hunk 2 above)
Comment (SylvainGuittard) [inline app/config/ezplatform.yml]: Very good point. Thank you

File: app/Resources/views/row/download_section.html.twig (hunk 1 above)
Comment (clash82) [inline app/Resources/views/row/download_section.html.twig]: try this:

```
{{ ez_field_value(release, 'file').fileName }}
```

File: web/assets/images/download.svg (hunk 13 above)
Comment (SylvainGuittard) [inline web/assets/images/download.svg]: I wanted to do it during https://jira.ez.no/browse/COM-19900
But let's do that now.

File: app/Resources/views/row/download_section.html.twig (hunk 1 above)
Comment (SylvainGuittard) [inline app/Resources/views/row/download_section.html.twig]: Thanks :)

**Comment by SylvainGuittard** [comment]
Final review @clash82

File: app/Resources/views/row/download_section.html.twig [hunk 17]
```diff
@@ -0,0 +1,150 @@
...
+                                            <a href="{{ ez_field_value(release, 'file') }}">{{ ez_render_field(release, 'release_version') }}</a>
+                                        </th>
+                                        <td class="file">
+                                            <a href="{{ ez_field_value(release, 'file') }}">
+                                                {{ ez_field_value(release, 'file').fileName }}
+                                            </a>
+                                        </td>
+                                        <td class="date">
+                                            <a href="{{ ez_field_value(release, 'file') }}">
+                                                {{ ez_field_value(release, 'release_date')|date("d M Y") }}
```
Comment (clash82) [inline app/Resources/views/row/download_section.html.twig]: single quotation marks

**Comment by clash82** [comment]
Those new SVG files looks very cool now :)
Besides the comment above +1

**Comment by SylvainGuittard** [comment]
Thanks for your review @clash82. I appreciate.
Ready to merge

**Pull request merged**